src/src/components/notification/Notification.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+<div
+  v-if="notification.show"
+  class="notification-container">
+  <div
+    class="notification-bubble"
+    :class="alertCss">
+      <fa 
+        icon="sort-up"
+        class="notification-top-icon" />
+      <div class="notification-bubble-content">
+          <fa :icon="iconChoice" />
+          {{ notification.alertText }}
+          <button 
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 12 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
        <button 
```